HMR added

kapilpaul opened the pull request:
HOT module replacement added and some necessary changes.

Commit:
hot module added

Commit:
hot module js and css loading and readme update

Commit:
composer.json and plugin.php name updated

Commit:
restructured files

Commit:
installer class added

tareq1988 commented:
You've taken my plugin and changed the author name, coding standard and everything. Why did you do that? The coding standard is not even WordPress coding standard.

kapilpaul commented:
I am really very sorry for that. I was doing some tests and added Hot Module Replacement with that plugin. I was not aware of the coding standards for WordPress. I am just watching your tutorial about coding standards and updated the plugin in my repo. So sorry, for removing the author name. It was not intended. I am updating it now.  Thank you.
